jest.config.base.js

```diff
@@ -3,6 +3,7 @@
 module.exports = {
   globals: {
     'ts-jest': {
+      isolatedModules: true,
```

Comment:
Is this related to the react version detection?

Reply:
Oh, shoot, sorry - didn't realize I'd left this in 😅. Removing.